src/test/python/test_stl_reader.py

```diff
@@ -29,32 +29,268 @@
 
 import unittest
 import ttconv.stl.reader
+import ttconv.style_properties as styles
+from ttconv.time_code import SmpteTimeCode, FPS_25
 
 class STLReaderTests(unittest.TestCase):
 
-  def test_irt_requirement_0164_001(self):
-    with open("src/test/resources/stl/irt/requirement-0164-001.stl", "rb") as f:
-      ttconv.stl.reader.to_model(f)
+  def test_irt_requirement_0061_001(self):
+    '''Testing Time Code In with value media 25 frames lowerbounds'''  
+    with open("src/test/resources/stl/irt/requirement-0061-001.stl", "rb") as f:
+      doc = ttconv.stl.reader.to_model(f)
+      p = doc.get_body().first_child().first_child()
+      self.assertEqual(p.get_begin(),\
+                      SmpteTimeCode.parse("00:00:00:00",FPS_25).to_temporal_offset())
+   
+  def test_irt_requirement_0061_004(self):
+    '''Testing Time Code In with value media 25 frames upperbounds'''
+    with open("src/test/resources/stl/irt/requirement-0061-004.stl", "rb") as f:
+      doc = ttconv.stl.reader.to_model(f)
+      p = doc.get_body().first_child().first_child()
+      self.assertEqual(p.get_begin(),\
```

Comment:
```suggestion
      self.assertEqual(p.get_begin(),
```

No need for `\` at the end of a line.

Reply:
True! Thanks!